internal/build/gen_index.sh

```
@@ -0,0 +1,62 @@
+		n="${d#$g/}"
+		if [ "x$n" != "x$d" ]; then
+			echo "$n" | tr '/' '-'
+			return
+		fi
+	done
+
+	echo "$d" | tr '/' '-'
+}
+
+mod_replace() {
+	local d="$1"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)